x/stake/keeper/distribute_sidechain.go

```diff
@@ -130,7 +133,13 @@ func (k Keeper) DistributeInBreathBlock(ctx sdk.Context, sideChainId string) {
 	for _, validator := range validators {
 		distAccCoins := k.bankKeeper.GetCoins(ctx, validator.DistributionAddr)
 		totalReward := distAccCoins.AmountOf(bondDenom)
-		totalRewardDec := sdk.ZeroDec()
+		totalRewardDec := sdk.NewDec(totalReward)
+		if sdk.IsUpgrade(sdk.BEPHHH) && sideChainId != MockSideChainIDForBeaconChain {
```

Comment:
Please try to transfer the `acumulatedFeeToBC` amount of funds to a system address. Then re-fetch the balance of `DistributionAddr` will be good enough.